src/backends/utils.js

```diff
@@ -0,0 +1,37 @@
+'use strict'
+
+const _ = require('lodash')
+const config = require('../services/config')
+
+function getCurrencyPair (source_ledger, destination_ledger) {
+  if (!_.isEmpty(config.get('tradingPairs'))) {
+    for (let pair of config.get('tradingPairs')) {
+      // trading pair is of the form
+      // ["<currency@<source_ledger>","<currency>@<destination_ledger>"]
+      if (pair[0].indexOf(source_ledger) === 4 &&
+        pair[1].indexOf(destination_ledger) === 4) {
+        return [pair[0].slice(0, 3), pair[1].slice(0, 3)]
+      }
+    }
+  }
+  // No currency pair found
+  return ['', '']
+}
+
+/**
+ * Check if we trade the given pair of assets
+ *
+ * @param {String} source The URI of the source ledger
+ * @param {String} destination The URI of the destination ledger
+ * @return {boolean}
+ */
+function hasPair (currencyPairs, source, destination) {
+  const currencyPair = getCurrencyPair(source, destination)
+  return _.includes(currencyPairs, currencyPair[0]) &&
```

Comment:
This method was moved from the fixer.io backend: https://github.com/interledger/five-bells-connector/pull/124/files#diff-732a4256af7e30fb991b56df0f8474f2L77. Both the fixerio and ilp-quoter backends had the same method, implemented slightly differently, so I consolidated them in one file.

This method first looks up the currency pair for the `source`/`destination` ledger (something like `['USD', 'EUR']`). `currencyPairs` is an array of the currencies the ledger supports. So we return `true` iff `currencyPairs` contains both the currencies the user wants to trade.

The variable names are probably the confusing part. I'll update them.